client/extensions/wp-super-cache/app/main.jsx

```diff
@@ -92,6 +104,7 @@ class WPSuperCache extends Component {
 					redirectUrl={ redirectUrl }
 				/>
 				<QueryStatus siteId={ siteId } />
+				{ this.trackPageView() }
```

Comment:
I'm not a big fan of the `render*` functions 🙁 

You can do instead
```jsx
<PageViewTracker
	path={ `/extensions/wp-super-cache/${ tab ? tab + '/' : '' }:site }
	title{ `WP Suprt Cache > ${ tab ? titlecase( tab ) : 'Easy' }` }
/>
```

Reply:
It looked to me a bit cluttered, but if you think that's clear enough, then sure! 👍

Reply:
I'm just morally and ethically opposed to the `render*` functions. Smells like your component could be split into smaller components to me. But it's up to you to change it or not 🙂